File: ApiSuvesaPos/Negocio/Modelo/cxp/compras/compras.vb
Namespace Modelo
    Public Class compras
        Public Property Id_Compra As Long
        Public Property Factura As Decimal
        Public Property CodigoProv As Integer
        Public Property SubTotalGravado As Decimal
        Public Property SubTotalExento As Decimal
        Public Property Descuento As Decimal
        Public Property Impuesto As Decimal
        Public Property TotalFactura As Decimal
        Public Property Fecha As DateTime
        Public Property Vence As DateTime
        Public Property FechaIngreso As DateTime
        Public Property MotivoGasto As String
        Public Property Compra As Boolean
        Public Property Contabilizado As Boolean
        Public Property Consignacion As Boolean
        Public Property Asiento As Decimal
        Public Property ContaInve As Boolean
        Public Property AsientoInve As Decimal
        Public Property TipoCompra As Char
        Public Property CedulaUsuario As String
        Public Property Cod_MonedaCompra As Integer
        Public Property FacturaCancelado As Boolean
        Public Property Gasto As Boolean
        Public Property TipoCambio As Decimal
        Public Property CambioImpuesto As Decimal
        Public Property Taller As Boolean
        Public Property Mascotas As Boolean
        Public Property num_orden As Decimal
        Public Property FEC As Boolean
        Public Property CodigoActividad As String
        Public Property EnviadoDGT As Boolean
        Public Property EstadoDGT As String
        Public Property ConsecutivoDGT As String
        Public Property ClaveDGT As String
        Public Property Trans As Boolean
        Public Property NumTrans As String
        Public Property Prepagada As Boolean

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxp/compras/compras.cs
using System;
using System.Collections.Generic;

namespace Modelo
{
    public class compras
    {
        public long Id_Compra { get; set; }
        public decimal Factura { get; set; }
        public int CodigoProv { get; set; }
        public decimal SubTotalGravado { get; set; }
        public decimal SubTotalExento { get; set; }
        public decimal Descuento { get; set; }
        public decimal Impuesto { get; set; }
        public decimal TotalFactura { get; set; }
        public DateTime Fecha { get; set; }
        public DateTime Vence { get; set; }
        public DateTime FechaIngreso { get; set; }
        public string MotivoGasto { get; set; }
        public bool Compra { get; set; }
        public bool Contabilizado { get; set; }
        public bool Consignacion { get; set; }
        public decimal Asiento { get; set; }
        public bool ContaInve { get; set; }
        public decimal AsientoInve { get; set; }
        public char TipoCompra { get; set; }
        public string CedulaUsuario { get; set; }
        public int Cod_MonedaCompra { get; set; }
        public bool FacturaCancelado { get; set; }
        public bool Gasto { get; set; }
        public decimal TipoCambio { get; set; }
        public decimal CambioImpuesto { get; set; }
        public bool Taller { get; set; }
        public bool Mascotas { get; set; }
        public decimal num_orden { get; set; }
        public bool FEC { get; set; }
        public string CodigoActividad { get; set; }
        public bool EnviadoDGT { get; set; }
        public string EstadoDGT { get; set; }
        public string ConsecutivoDGT { get; set; }
        public string ClaveDGT { get; set; }
        public bool Trans { get; set; }
        public string NumTrans { get; set; }
        public bool Prepagada { get; set; }
        public List<Modelo.articulos_comprados> Articulos_Comprados { get; set; } = new List<Modelo.articulos_comprados>();
        public List<Modelo.articulos_gastos> Articulos_Gastos { get; set; } = new List<Modelo.articulos_gastos>();
    }
}
